components/motor/gpio/motor_encoder.go

```diff
@@ -467,7 +475,7 @@ func (m *EncodedMotor) position(ctx context.Context, extra map[string]interface{
 	if err != nil {
 		return 0, err
 	}
-	return ticks, nil
+	return ticks + m.offsetInTicks, nil
```

Comment:
you could use extra or change the signature in the `position` function to pass this in from the `ResetZeroPosition` function rather than add it to the motor struct, since it's not doing anything r checked anywhere else.

Reply:
I think the issue with using extra is that it's not saved for every time rpm monitor calls position. so I think the offset would just fully be lost

Reply:
I tried that at first but was having issues with the offset staying over multiple motor commands